src/python/espressomd/thermostat.pyx

```diff
@@ -306,13 +309,13 @@ cdef class Thermostat(object):
                 ELSE:
                     langevin_gamma_rotation = langevin_gamma
 
-        global thermo_switch
-        thermo_switch = (thermo_switch | THERMO_LANGEVIN)
-        mpi_bcast_parameter(FIELD_THERMO_SWITCH)
         mpi_bcast_parameter(FIELD_TEMPERATURE)
         mpi_bcast_parameter(FIELD_LANGEVIN_GAMMA)
         IF ROTATION:
             mpi_bcast_parameter(FIELD_LANGEVIN_GAMMA_ROTATION)
+        global thermo_switch
+        thermo_switch = (thermo_switch | THERMO_LANGEVIN)
+        mpi_bcast_parameter(FIELD_THERMO_SWITCH)
```

Comment:
Pointless change

Reply:
@hmenke could you please clarify this. Maybe I've missed how correctly `thermo_switch` should set for the given thermostat within the latest architecture. Similar code snippets exist for other thermostat types in the same file.

Reply:
Well, it looks like you just moved those three lines a little down.

Reply:
Oops, reverted.